src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c

```diff
@@ -776,6 +776,26 @@ static int xclmgmt_read_subdev_req(struct xclmgmt_dev *lro, char *data_ptr, void
 		xclmgmt_subdev_get_data(lro, subdev_req->offset,
 			subdev_req->size, resp, &current_sz);
 		break;
+	case XCL_SDR_BDINFO:
+		current_sz = 4 * 1024;
```

Comment:
Can we give '4' and '1024' a meaningful macro name?

Reply:
i will update, thank you.

Reply:
Sorry that I was not clear. I mean we should define macros for these constant values so that we can get better readability and easy to maintain. Also, where do we check if the size of SDR data obtained does not go beyond this current_sz?

Reply:
Hi @maxzhen  Actually, there is a pending AI to get the size of the sensor data first and then issue the request for actual sensor data. So, we are just using 4k as size for now till we fix it.